modules/ROOT/nav.adoc

```diff
@@ -19,14 +19,14 @@
 * xref:howtos:distributed-acid-transactions-from-the-sdk.adoc[Distributed ACID Transactions]
 * xref:howtos:transcoders-nonjson.adoc[Transcoders & Non-JSON]
 * xref:howtos:working-with-collections.adoc[Working with Collections]
+* xref:howtos:encrypting-using-sdk.adoc[Encrypting Your Data]
```

Comment:
Note, the others are positioned alphabetically  ;-)

Reply:
[Learn and Advanced Data Operations are the only 2 sections in which that is reasonably sensible to do, btw]